example/src/main/java/io/netty/example/http2/helloworld/server/Http2ServerInitializer.java

```diff
@@ -49,9 +50,18 @@ public UpgradeCodec newUpgradeCodec(String protocol) {
     };
 
     private final SslContext sslCtx;
+    private final int maxHttpContentLength;
 
     public Http2ServerInitializer(SslContext sslCtx) {
+        this(sslCtx, 16 * 1024);
+    }
+
+    public Http2ServerInitializer(SslContext sslCtx, int maxHttpContentLength) {
+        if (maxHttpContentLength < 0) {
```

Comment:
@normanmaurer - I think this is OK (you could expect 0 content) and is inline with `MessageAggregator.validateMaxContentLength`. However the exception message needs to be updated.